circle.yml

```diff
@@ -6,11 +6,25 @@ machine:
     # DB config. Using default CircleCI's database.
     TERMINUS_ENV: ci-$CIRCLE_BUILD_NUM
     TERMINUS_SITE: wp-native-php-sessions
+    WORDPRESS_ADMIN_USERNAME: pantheon
+    # Somehow in the first repo to use a global variable for the admin password
+    # it was possible to set a random password at this stage of CircleCI and
+    # the same random password would persist for all usages.
+    # https://github.com/pantheon-systems/pantheon-wordpress-upstream-tests/blob/1da29d6f0ac1fa469c1567ed8a5a7e0949d359c7/circle.yml#L8
+    # However in this repo, that is not working. Either the openssl command is
+    # not executed and the value is taken as a string
+    # https://github.com/pantheon-systems/wp-native-php-sessions/commit/9911b256d634483f81dea08a3f85a8351825b3bc
+    # Or the command is reexecuted on each usage.
+    # https://github.com/pantheon-systems/wp-native-php-sessions/commit/2630b26aed79d98271857d4c61f62194407b6184
+    # So this workaround is to save the password to a file in dependencies:pre
+    # and then read that file over and over.
+    WORDPRESS_ADMIN_PASSWORD: $(cat ~/WORDPRESS_ADMIN_PASSWORD)
```

Comment:
Blerg, I just figured it out. In the original usage, the installation of WordPress (where the password is used for the first time) and all the subsequent usages happen in the same CircleCI process: `- ./test.sh` https://github.com/pantheon-systems/pantheon-wordpress-upstream-tests/blob/master/circle.yml#L28